x/axelarnet/keeper/msg_server.go

```diff
@@ -154,7 +154,7 @@ func (s msgServer) ConfirmDeposit(c context.Context, req *types.ConfirmDepositRe
 
 	}
 
-	transferID, err := s.nexus.EnqueueForTransfer(ctx, depositAddr, amount, s.GetTransactionFeeRate(ctx))
```

Comment:
The `TransactionFeeRate` param in `axelarnet/evm` is not needed anymore. What's the appropriate way to deprecate/remove it? Just delete the reference to it or should it be part of the state migration?

Reply:
I think to reserve/deprecate it is enough